common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -385,14 +389,27 @@ private static CommandResult executeCommand(@SuppressWarnings(WarningType.rawtyp
                         command.getParameters().getCorrelationId());
             }
         } else /* baseException == null */ {
-            if (result != null && result instanceof AcquireTokenResult) {
+            if (result instanceof AcquireTokenResult) {
                 //Handler handler, final BaseCommand command, BaseException baseException, AcquireTokenResult result
                 commandResult = getCommandResultFromTokenResult(baseException, (AcquireTokenResult) result,
                         command.getParameters().getCorrelationId());
             } else {
+
                 //For commands that don't return an AcquireTokenResult
                 commandResult = new CommandResult(CommandResult.ResultStatus.COMPLETED, result,
                         command.getParameters().getCorrelationId());
+
+                //If any record needs a refresh, those records need to be refreshed.
+                if(result instanceof List<?>
```

Comment:
nit: at runtime, java knows only raw types, so there's no point checking for List<?>

Reply:
Removed unnecessary check.